compose/ui/ui/src/webTest/kotlin/androidx/compose/ui/window/MouseEventsTest.kt

```diff
@@ -18,24 +18,73 @@ package androidx.compose.ui.window
 
 import androidx.compose.foundation.ExperimentalFoundationApi
 import androidx.compose.foundation.PointerMatcher
+import androidx.compose.foundation.background
 import androidx.compose.foundation.layout.Box
 import androidx.compose.foundation.layout.fillMaxSize
 import androidx.compose.foundation.onClick
+import androidx.compose.material.Text
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.setValue
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.OnCanvasTests
+import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.input.pointer.PointerButton
 import androidx.compose.ui.input.pointer.PointerEvent
 import androidx.compose.ui.input.pointer.PointerEventType
 import androidx.compose.ui.input.pointer.pointerInput
 import kotlin.test.Test
 import kotlin.test.assertEquals
+import kotlinx.browser.document
+import kotlinx.browser.window
+import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.NonCancellable.isActive
+import kotlinx.coroutines.delay
 import kotlinx.coroutines.test.runTest
+import kotlinx.coroutines.withContext
 import org.w3c.dom.events.MouseEvent
 import org.w3c.dom.events.MouseEventInit
 
 class MouseEventsTest : OnCanvasTests {
 
+    @Test
+    fun createMouseEvent() = runTest {
```

Comment:
this test shows very weird behaviour in k/js:
The offsetX and offsetY is smaller by exactly 8 than the expected value. No such thing with k/wasm.